content/docs/quickstart.mdx

```diff
@@ -22,74 +22,198 @@ keywords:
   ]
 ---
 
-import ConfigDocker from '@site/content/examples/config/config.docker.yaml.md';
-import DockerCompose from '@site/content/examples/docker/basic.docker-compose.yml.md';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-# Run Pomerium Core With Docker
+# Pomerium Zero Quickstart
 
-Run Pomerium Core with Docker containers in **under 5 minutes**.
+In this Quickstart, you'll install and run Pomerium Zero in a Docker container. By the end, you'll have a **personal account** and cluster where you can configure routes and policies to your own apps and services.
 
-The Core quickstart uses Pomerium's [**Hosted Authenticate Service**](/docs/capabilities/hosted-authenticate-service), but you can also configure a [**self-hosted authenticate service**](/docs/capabilities/self-hosted-authenticate-service) to integrate with Pomerium.
+## Before you start
 
-## Prerequisites
+- Install [Docker](https://docs.docker.com/install/) and [Docker Compose](https://docs.docker.com/compose/install/)
+- Create a [**free Pomerium Zero account**](https://console.pomerium.app/create-account)
 
-[Docker] and [Docker Compose]
+## Get Pomerium Zero configuration
 
-## Configure Pomerium
+In the onboarding console, select the **Docker** tab and copy the Docker Compose configuration.
 
-Create a `config.yaml` file in the root of your project.
+Paste the configuration in a `docker-compose.yaml` file:
 
-Add the configuration below to `config.yaml`:
+```yaml title="docker-compose.yaml"
+services:
+  pomerium:
+    image: pomerium/pomerium:v0.25.1
+    ports:
+      - 80:80
+      - 443:443
+    restart: always
+    environment:
+    // highlight-start
+      POMERIUM_ZERO_TOKEN: <REPLACE_WITH_YOUR_TOKEN>
+    // highlight-end
+      XDG_CACHE_HOME: /var/cache
+      AUTOCERT_DIR: /var/cache/autocert
+    volumes:
+      - pomerium-cache:/var/cache
 
-<ConfigDocker />
+volumes:
+  pomerium-cache:
+```
 
-## Configure Docker
+Replace the value of `POMERIUM_ZERO_TOKEN` with the one generated for you in the onboarding config. The token links this **Pomerium Zero Cluster** with your account.
 
-Create a `docker-compose.yaml` file in the root of your project.
+<details>
+  <summary>Clusters Overview</summary>
+  <div>Include high-level summary of Clusters concept in PZ. Link to Clusters documentation.</div>
+</details>
 
-Add the configuration below to `docker-compose.yaml`:
+## Run Docker Compose
 
-<DockerCompose />
+```bash
+docker compose up -d
+```
 
-## Run Docker Compose
+Pomerium Zero will attempt to detect your **public IP** address to connect you to your cluster.
+
+But, it won't be able to because you're running Pomerium Zero in an isolated container with its own private network and IP address.
```

Comment:
I think right after this we need to showcase what someone is seeing so it's spelled out for them

"You'll see an error like the following (see attached image), and that's completely normal!"